### DISCUSSION TITLE
How to make a custom hook for pytest_runtest_makereport run after internal hooks?

### DISCUSSION THREAD

**Discussion by njgrisafi** [pytest-dev/pytest#11681] [General]
Hello!

I'm trying to write a simple hookwrapper for `pytest_runtest_makereport`.

I want my wrapper to run after the one defined in `skipping`: https://github.com/pytest-dev/pytest/blob/a536f49d910eda84cbd0608e6edb1b70bb0868b2/src/_pytest/skipping.py#L265-L292

This is because I want to absorb the xfail information from the report.

I've tried the following with no luck
```python
@pytest.hookimpl(hookwrapper=True)
def pytest_runtest_makereport(item: Item, call: CallInfo):
    ...
```
and
```python
@pytest.hookimpl(hookwrapper=True, trylast=True)
def pytest_runtest_makereport(item: Item, call: CallInfo):
    ...
```

Really appreciate any help, thank you!

**Comment by bluetech** [comment]
Teardowns are run in reverse order -- so can you try `tryfirst=True` instead of `trylast=True` and see if it does the trick?
